contracts/simpleKitty/URIMetadata.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity 0.4.24;
+
+contract URIMetadata {
+    function tokenURI(uint256 _tokenId) external view returns (string) {
```

Comment:
does it make sense to check that such token ID exists?

Reply:
Yes, I think it makes sense. Added check https://github.com/poanetwork/cryptokitties-xdai-demo/pull/8/commits/3233812ab92e8ca171e2fb09ee0994b6b07bed5f